css/ladders-landing.css

```diff
@@ -0,0 +1,98 @@
+.ladder {
+  height: 35vh;
+  position: relative;
+  text-align: center;
+  border: 1px solid #000;
+  border-radius: 1rem;
+  box-sizing:border-box;
+  width:100%;
+  padding: 0.6rem;
+  font-family: 'Kranky', cursive;
+  background-color: white;
+}
+
+.ladder--new .ladder__image {
+  opacity: 0.6;
+}
+
+.ladder__title {
+  margin-top:1rem;
+  font-size: 4rem;
+}
+
+.ladder__add {
+  font-size: 30rem;
+  margin: 0;
+  width: 90%;
+  position: absolute;
+  top: 60%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+
+}
+
+.ladder__image {
+  width: 90%;
+  position: absolute;
+  top: 60%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  max-height: 70%
+}
+
+/* MODAL */
+
+.modal {
+  display: block;
+  width: 80vw;
+  height: 60vh;
+  position: fixed;
+  z-index: 100;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+  background: white;
+  box-shadow: 0 0 60px 10px rgba(0, 0, 0, 0.9);
+}
+.closed {
```

Comment:
because the class just sets display to none, is it alright to keep it as closed?